docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md

```diff
@@ -28,20 +28,20 @@ The following list provides the Kaldi-specific parameters.
 Examples of CLI Commands
 ########################
 
-* To launch Model Optimizer for the ``wsj_dnn5b_smbr`` model with the specified ``.nnet`` file:
+* To launch model conversion for the ``wsj_dnn5b_smbr`` model with the specified ``.nnet`` file:
    
   .. code-block:: cpp
 
     mo --input_model wsj_dnn5b_smbr.nnet
```

Comment:
Applicable here and everything else in the example: we need both Python API examples and CLI examples. If they are prepared in a separate PR, it should be merged with the current PR and we need to continue working on them in one shot. It is hard to track multiple changes in the same document pages independently.

Please collaborate with @andrei-kochin, @rkazants, @popovaan and @kblaszczak-intel to make aligned changes. Looks like we have only 2 weeks left for the documentation improvement.

Reply:
@slyalin @rkazants 
Do we want to add Python API commands for tutorials like "Convert_Model_From_Kaldi.md"?
These tutorials may contain commands that do not work and we do not have resources to check them all.

Reply:
The Python API and CLI example commands for the main documentation have been introduced with the following commit:
https://github.com/openvinotoolkit/openvino/pull/17454/commits/d4c2cdfac6fd12a25f10c75c54129967af2465e1 
(picked from: https://github.com/openvinotoolkit/openvino/pull/17203)

I believe the #17203 can be closed and further changes regarding 'MO documentation reorganization` propagated within the scope of this PR.

Reply:
> @slyalin @rkazants Do we want to add Python API commands for tutorials like "Convert_Model_From_Kaldi.md"? These tutorials may contain commands that do not work and we do not have resources to check them all.

As we deprecated those legacy FEs, I don't think that we need to spend time on it. Except for common pages not solely dedicated to those FEs where not doing that will lead to inconsistency. Another question is why should we have such examples on common pages, I think we can remove such examples and make the main flow free from Kaldi, Caffe and MXNet examples. As the result we won't have any examples migrated to Python for those deprecated FEs.

Reply:
> > @slyalin @rkazants Do we want to add Python API commands for tutorials like "Convert_Model_From_Kaldi.md"? These tutorials may contain commands that do not work and we do not have resources to check them all.
> 
> As we deprecated those legacy FEs, I don't think that we need to spend time on it. Except for common pages not solely dedicated to those FEs where not doing that will lead to inconsistency. Another question is why should we have such examples on common pages, I think we can remove such examples and make the main flow free from Kaldi, Caffe and MXNet examples. As the result we won't have any examples migrated to Python for those deprecated FEs.

Implemented the following changes with the [0cd5e32](https://github.com/openvinotoolkit/openvino/pull/17454/commits/0cd5e329aa349a6b9455897daf7cae52020648bc ) commit: